provisioning/miniprov/hare_mp/main.py

```diff
@@ -225,6 +227,42 @@ def init(args):
         raise RuntimeError(f'Error while initializing cluster :key={error}')
 
 
+def start_mkfs(hostname: str, hare_config_dir: str):
+    motr_config_dir = '/etc/motr'
+    os.makedirs(motr_config_dir, exist_ok=True)
+    shutil.copy(f'{hare_config_dir}/confd.xc', motr_config_dir)
+    src = f'{hare_config_dir}/sysconfig/motr/{hostname}'
+    for file in os.listdir(src):
+        shutil.copy(os.path.join(src, file),'/etc/sysconfig')
+    MOTR_NODE_UUID = str(uuid.uuid4())
+    contents = f"MOTR_NODE_UUID='{MOTR_NODE_UUID}'"
```

Comment:
better, let's just use /etc/machine-id as MOTR_NODE_UUID, if /etc/machine-id is not present then only generate a uuid.

Reply:
done

Reply:
@Shreya-18, just for testing purpose, did you try the failure case where /etc/machine-id is not present?

Reply:
/etc/machine-id will be required in prepare and config stages too, so may be its a good idea to check if /etc/machine-id exists and create if not. Can you move this code to post-install section?

Reply:
shifted to setup_logging() since its the first place where machine-id is used

Reply:
done